WORKSPACE

```diff
@@ -15,6 +15,40 @@ load("@io_bazel_rules_closure//closure:defs.bzl", "closure_repositories")
 
 closure_repositories()
 
+http_archive(
+    name = "io_bazel_rules_go",
+    sha256 = "ef1aa6a368808d3aa18cbe588924f15fb8fac75d80860080355595e75eb9a529",
+    strip_prefix = "rules_go-0.4.0",
+    urls = [
+        "https://github.com/bazelbuild/rules_go/archive/0.4.0.tar.gz",
```

Comment:
I recommend *always* listing the bazel-mirror URL first, even if it's a dead link.